onnxruntime/core/providers/cpu/tensor/transpose.cc

```diff
@@ -15,7 +15,7 @@ namespace onnxruntime {
 
 // ComputeOffset: compute offset into a tensor. This is essentially the dot-product of
 // index and stride, restricted to the specified number of axes.
-size_t ComputeOffset(const std::vector<int64_t>& index, const std::vector<size_t>& stride, int64_t num_axes) {
+inline size_t ComputeOffset(const std::vector<int64_t>& index, const std::vector<size_t>& stride, int64_t num_axes) {
   size_t offset = 0;
```

Comment:
static

Reply:
I think static inline is a good choice.

---
In reply to: [264967422](https://github.com/Microsoft/onnxruntime/pull/561#discussion_r264967422) [](ancestors = 264967422)